src/visualization/components/Snapshot.tsx

```diff
@@ -49,7 +51,7 @@ class Snapshot extends React.Component<any, any> {
       for (var i = startPointer; i < events.length; i++) {
         var event = events[i];
         if (event.time <= time) {
-          let parser = event.type === "Layout" && this.props.boxmodel ? "BoxModel" : event.type; 
+          let parser = event.type === "Layout" && this.props.view == 1 ? "BoxModel" : event.type; 
```

Comment:
would be good to make this an enum so that the tab values and these remain consistent when we add more options